### ISSUE TITLE
niuwan1

### ISSUE THREAD

**Issue by niuwan1** [andruekonst/egbm-gam#1]
Hi @andruekonst,

Could you please release the classification version of EGBM? I tried to implement the classification version and modified the loss function of GBMs and MSE loss into cross-entropy to compute residuals. Also, I increase the pretraining iterations of each GBM to separate the samples by a hyperplane at the beginning of the training of EGBM as your paper referred. However, the results are not good. Could you please advise? 

Many thanks
